conda_build/exceptions.py

```diff
@@ -0,0 +1,46 @@
+import textwrap
+SEPARATOR = "-" * 70
+
+indent = lambda s: textwrap.fill(textwrap.dedent(s))
+
+
+class CondaBuildException(Exception):
```

Comment:
Conda catches RuntimeError and converts it into sys.exit. So any error that will be raised to the user that doesn't indicate a bug should subclass from RuntimeError.